crypto/fipsmodule/ec/ec_nistp.c

```diff
@@ -259,3 +259,44 @@ void ec_nistp_point_add(const ec_nistp_meth *ctx,
   cmovznz(z3, ctx->felem_num_limbs, z2nz, z1, z_out);
 }
 
+static int16_t get_bit(const EC_SCALAR *in, size_t in_bit_size, size_t i) {
+  if (i >= in_bit_size) {
```

Comment:
Is constime-ness relevant to this function?

Reply:
not in the context it's used in, but even there it is used in a constant-time manner.